### ISSUE TITLE
locale support ?

### ISSUE THREAD

**Issue by waby38b** [BurntSushi/imdb-rename#6]
Do you plan to add an option to use the localised name and not the original title?

Exemple:
https://www.imdb.com/title/tt2294629/releaseinfo?ref_=tt_ov_inf

Original Title (default) : Frozen
=> with option "French":  La reine des neiges

**Comment by BurntSushi** [comment]
No, I don't have any plans to do this myself. If a _simple_ code change can do this then I'd be happy to accept a PR.
